variables.tf

```diff
@@ -12,3 +12,9 @@ variable "source_account_role_names" {
   description = "The name of the role that the assume role call will be coming from. Again, this should correspond to a group."
   type        = list
 }
+
+variable "mfa_present" {
+  description = "Whether the created policy will include MFA."
+  type        = string
```

Comment:
Seems like it would be better to make this a `bool` and then just cast it to a string in the policy (see above)

https://www.terraform.io/docs/configuration/functions/tostring.html